src/main/java/de/qabel/desktop/storage/FolderNavigation.java

```diff
@@ -39,9 +39,7 @@ protected void uploadDirectoryMetadata() throws QblStorageException {
 	public DirectoryMetadata reloadMetadata() throws QblStorageException {
 		logger.info("Reloading directory metadata");
 		// duplicate of navigate()
-		try {
-			StorageDownload download = readBackend.download(dm.getFileName(), directoryMetadataMHashes.get(Arrays.hashCode(dm.getVersion())));
-
+		try (StorageDownload download = readBackend.download(dm.getFileName(), directoryMetadataMHashes.get(Arrays.hashCode(dm.getVersion())))) {
```

Comment:
`directoryMetadataMHashes.get(Arrays.hashCode(dm.getVersion())))` could be moved into a seperate method.